front/components/app/NewBlock.tsx

```diff
@@ -27,16 +27,10 @@ export default function NewBlock({
     name: string;
     description: string;
   }[] = [
-    {
```

Comment:
Bullish to remove it but that means removing it from the docs as well. This should be in a separate PR IMHO

Reply:
Make sense, will do